qbraid/runtime/schemas/experiment.py

```diff
@@ -129,3 +129,13 @@ class QuEraQasmSimulationMetadata(GateModelExperimentMetadata):
     flair_visual_version: Optional[str] = Field(None, alias="flairVisualVersion")
     atom_animation_state: Optional[dict[str, Any]] = Field(None, alias="atomAnimationState")
     logs: Optional[list[dict[str, Any]]] = None
+
+
+class AnnealingExperimentMetadata(BaseModel):
+    """Metadata specific to annealing experiments."""
+
+
+class NECVectorAnnealerMetadata(AnnealingExperimentMetadata):
+    """Result data specific to jobs submitted to the NEC Vector Annealer."""
+
+    job_data: list[dict[str, Any]] = Field(alias="jobData")
```

Comment:
Can we choose a naming convention that is more descriptive than `job_data`? What does "job_data" represent?